src/main/java/com/sfuentes/daysLivedOfAPerson/DaysLivedOfAPerson.java

```diff
@@ -0,0 +1,38 @@
+package com.sfuentes.daysLivedOfAPerson;
+
+
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+
+import java.time.DateTimeException;
+import java.time.LocalDate;
+import java.time.temporal.ChronoUnit;
+
+@Getter
+@RequiredArgsConstructor
+public class DaysLivedOfAPerson {
+
+  private final String date;
+  long daysLived;
+
+  public long calculateDaysLived() {
+    var listOfStr = getDate().split("/");
+
+    LocalDate now = LocalDate.now();
+    LocalDate birthDate = LocalDate.of(
+        Integer.parseInt(listOfStr[0]),
+        Integer.parseInt(listOfStr[1]),
+        Integer.parseInt(listOfStr[2])
+    );
+
```

Comment:
You can use the method parse to avoid split and then use the values:

LocalDate birthDate = LocalDate.parse(getDate(), DateTimeFormatter.ofPattern("yyyy/MM/dd"));

Reply:
Fixed it.